build.gradle

```diff
@@ -307,7 +307,7 @@ dependencies {
     compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
     compile group: 'uk.gov.hmcts.reform', name: 'doc-assembly-client', version: '1.0.6'
     compile group: 'uk.gov.hmcts.reform', name: 'document-management-client', version: '7.0.0'
-    compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '4.1.33'
+    compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: 'DEV-SNAPSHOT'
```

Comment:
Can use version 4.1.35.RC-SSCS-8495-1

Reply:
should also keep sscs-pdf-email-common to 1.4.20-RC-SSCS-8495-1

Reply:
Yeh that was intentional as wanted to wait until 8495 was merged. I've now rebased my branch and created a new  4.1.35.RC-SSCS-8495-3 version